src/pages/nonprofit/application.tsx

```diff
@@ -1,9 +1,23 @@
-import { Flex } from "@chakra-ui/react";
+import { Flex, VStack, HStack, Link, Heading } from "@chakra-ui/react";
+import ApplicationCard from "src/components/ApplicationCard";
 
 function NonprofitApplicationPage() {
+  // TODO: pass in the current project's id
+  const projectId = "616ba502a1667f4a0eadcfa1";
+
   return (
-    <Flex height="100%" width="100%">
-      <Flex margin="auto">Fill Out Application</Flex>
+    <Flex height="100%" width="100%" marginTop={10}>
+      <VStack align="stretch" margin="auto">
+        <HStack spacing="90px">
```

Comment:
Some alignment issues with this HStack, see Figma design for how this should be placed above the application.